docs/third-party-developers/extensibility/data-store/query-state.md

```diff
@@ -0,0 +1,54 @@
+# Query State Store <!-- omit in toc -->
+
+## Table of contents <!-- omit in toc -->
+
+-   [Overview](#overview)
+-   [Usage](#usage)
+-   [Actions](#actions)
+    -   [setQueryValue( context, queryKey, value )](#setqueryvalue-context-querykey-value-)
+    -   [setValueForQueryContext( context ,value )](#setvalueforquerycontext-context-value-)
+
+## Overview
```

Comment:
Can we figure out and add some real-life use cases for why devs might want to interact with this store?